packages/utils/src/index.js

```diff
@@ -63,6 +63,8 @@ export { mixins };
  * @param {number} [dimensions=2] The number of coordinates per vertex in the input array
  * @return {number[]} Triangulated polygon
  */
+import './settings';
```

Comment:
Could you maybe move this to before the `./browser` export. This is weird because the earcut doc is for the earcut export.